models.py

```diff
@@ -0,0 +1,19 @@
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+
+class Profile(db.Model):
+    email_id = db.Column(db.String(250), unique=True, nullable=False, primary_key=True)
```

Comment:
create a primary key using an ID, instead of an email address. The email address should only be used as a unique identifier.